org.eclipse.scout.rt.server.commons.test/src/test/java/org/eclipse/scout/rt/server/commons/context/HttpRunContextProducerTest.java

```diff
@@ -40,6 +46,23 @@ public class HttpRunContextProducerTest {
   static final Locale TEST_LOCALE = Locale.CANADA_FRENCH;
   static final Subject TEST_SUBJECT = BEANS.get(ServletFilterHelper.class).createSubject(BEANS.get(SimplePrincipalProducer.class).produce(TEST_SUBJECT_NAME));
 
+  protected static IBean<?> s_accessControlMockBean;
```

Comment:
- change to non-static and use `@BeanMock`
- remove cleanup method
- change setup to just contain `when(accessControlMock.getUserIdOfCurrentSubject()).thenReturn(TEST_SUBJECT_NAME);`